src/pages/subreddit/Subreddit.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useEffect, useState} from 'react';
+import {Link, useParams} from "react-router-dom";
+import axios from "axios";
+import Header from "../../components/header/Header";
+import './Subreddit.css';
+import { ReactComponent as BackIcon } from '../../assets/back.svg';
+
+
+function Subreddit() {
+    const {id} = useParams();
+    const [catchError, setCatchError] = useState(null);
+    const [isLoading, setLoading] = useState(false);
+    const [details, setDetails] = useState({});
+
+
+    useEffect(() => {
+        async function fetchData() {
+            setLoading(true);
+            setCatchError(false);
+            try {
+                const response = await axios.get(`https://www.reddit.com/r/${id}/about.json`);
+
+                setDetails(response.data.data);
+            } catch (e) {
+                console.error(e);
+                setCatchError(true);
+            }
+            setLoading(false);
+
+        }
+
+        fetchData();
+
+    }, []);
+
+    console.log(details);
+
+    return (
+
+        <>
+            <Header>
+                {/*    deze nog tot comoponent maken met hoofdletter*/}
+                <h1>Title r/{id}</h1>
+                <h4>Subreddit specifitations</h4>
+
+
+            </Header>
+            <main className="outer-content-container subreddit">
+                {Object.keys(details).length > 0 && (
```

Comment:
Hier wordt die dan simpeler
```suggestion
                {details && (
```